### ISSUE TITLE
Valoración

### ISSUE THREAD

**Issue by lmorillas** [DWES-LE/streamlit-2---cuadro-de-mandos-personal-belrapa#3]
El readme tiene que ser sobre tu proyecto. Puedes eliminar lo que yo escribí. 
La tabla inicial mezcla datos (Aragón / España) y no aporta información significativa, no?
No veo un objetivo claro en el cuadro de mandos. No se trata de mostar información sin sentido, sino de intentar resolver un problema de visualización
Hay gráficos y comparativas interesantes que puedes conseguir con estos datos. 
Valoración propuesta 5

**Comment by belrapa** [comment]
Cierto, la primera tabla solo muestra todos los datos que existen, como una tabla referencia, donde después se puede comparar con las tablas que realice.
Lo del readme, no estaba seguro si lo podía retocar mucho, así que por si acaso no elimine lo tuyo.
El objetivo de las visualizaciones, es poder ver comparativas de las distintas columnas, como por ejemplo, la recaudación que se hizo por espectador, estableciendo los datos de la recaudación en los conciertos realizados en España en general con la recaudación conseguida en los conciertos de Aragón solamente, especificando si era conciertos de música popular o clásica.

Intentaba conseguir otras gráficas y comparativas, con mejores diseños y que quedaría bastante mejor para el proyecto, pero a la hora de implementarlo me daba errores, nombrando a los distintos con su respectivo nombre, y no se si seria por las importaciones.
Así que, por eso me decante por hacer esas graficas y comparativas, si no podría haber estado mejor.
